src/capella/presets/minimal.rs

```diff
@@ -0,0 +1,172 @@
+pub use crate::altair::minimal::SYNC_COMMITTEE_SIZE;
+pub use crate::altair::minimal::{
+    AggregateAndProof, Attestation, AttesterSlashing, ContributionAndProof, HistoricalBatch,
+    IndexedAttestation, LightClientUpdate, PendingAttestation, SignedAggregateAndProof,
+    SignedContributionAndProof, SyncAggregate, SyncCommittee, SyncCommitteeContribution,
+    SyncCommitteeMessage,
+};
+pub use crate::bellatrix::minimal::{BYTES_PER_LOGS_BLOOM, MAX_BYTES_PER_TRANSACTION, MAX_EXTRA_DATA_BYTES, MAX_TRANSACTIONS_PER_PAYLOAD, PROPORTIONAL_SLASHING_MULTIPLIER_BELLATRIX};
+pub use crate::bellatrix::minimal::Transaction;
+use crate::capella;
+use crate::capella::presets::Preset;
+pub use crate::phase0::minimal::{
+    EPOCHS_PER_HISTORICAL_VECTOR, EPOCHS_PER_SLASHINGS_VECTOR, ETH1_DATA_VOTES_BOUND,
+    HISTORICAL_ROOTS_LIMIT, MAX_ATTESTATIONS, MAX_ATTESTER_SLASHINGS, MAX_DEPOSITS,
+    MAX_PROPOSER_SLASHINGS, MAX_VALIDATORS_PER_COMMITTEE, MAX_VOLUNTARY_EXITS,
+    SLOTS_PER_HISTORICAL_ROOT, VALIDATOR_REGISTRY_LIMIT,
+};
+
+pub use capella::*;
+
+pub const MAX_WITHDRAWALS_PER_PAYLOAD: usize = 16;
+pub const MAX_BLS_TO_EXECUTION_CHANGES: usize = 16;
+pub const MAX_VALIDATORS_PER_WITHDRAWALS_SWEEP: usize =16384;
+
```

Comment:
some of these are off; see constants here: https://github.com/ethereum/consensus-specs/blob/dev/presets/minimal/capella.yaml

Reply:
Good catch! fixed now